interfaces/cython/cantera/test/test_convert.py

```diff
@@ -468,7 +468,13 @@ def test_extra(self):
         self.assertEqual(desc, 'This is an alternative description.')
         for key in ['foo', 'bar']:
             self.assertIn(key, yml.keys())
-
+    
+    def test_error_for_big_element_number(self):
+        try:
+            self.convert('big_ele_num_err_mech.txt', thermo='big_ele_num_err_thermo.txt',
+                        output='big_ele_num_err.yaml')
+        except Exception as e:
+            self.assertIn('the first line has an incorrect format', str(e))
```

Comment:
You can use the `self.assertRaisesRegex` context manager here, rather than the try/except block. Check out line 443 in this file for an example.

Reply:
You can also leave the extension off of the output file and the test class should be able to handle that.